stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -185,4 +194,30 @@ class HomeViewModel @Inject constructor(
             .atZone(ZoneId.systemDefault())
             .toLocalDateTime()
     }
+
+    @JvmName("getToolbarTitle1")
+    fun getToolbarTitle(): LiveData<TransactionType> {
+        return toolbarTitle
+    }
+
+    fun setToolbarTitle(transactionType: TransactionType) {
+        _toolbarTitle.value = transactionType
+    }
+
+    fun setSubtitle(from: String, to: String, type: TransactionType) {
```

Comment:
could you add a test to check this method, that is show the correct subtitle depending the transaction type, facility and deliver to?